mlflow/server/js/src/components/MetricsPlotPanel.js

```diff
@@ -521,7 +521,10 @@ const mapStateToProps = (state, ownProps) => {
   // Flat array of all metrics, with history and information of the run it belongs to
   // This is used for underlying MetricsPlotView & predicting chartType for MetricsPlotControls
   const metricsWithRunInfoAndHistory = _.flatMap(runUuids, (runUuid) => {
-    const runDisplayName = Utils.getRunDisplayName(getRunTags(runUuid, state), runUuid);
+    const runDisplayName = Utils.getRunDisplayName(
+      getRunTags(runUuid, state),
+      runUuid.slice(0, 6) + '...'
```

Comment:
What do you think about setting the run name wrapper to `text-overflow: ellipsis`? That way, we won't lose any information in the DOM.
https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
@Zangr Thanks for the comment! If we use `text-overflow`, do we need to set `max-width`?

Reply:
@Zangr It seems this can't be applied to the legends.